stories/browser/src/Sharing.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { View } from 'react-native';
+
+import { uiLocalized } from '@tonlabs/localization';
+import {
+    BubbleActionButton,
+    BubbleSimplePlainText,
+    ChatMessageType,
+    MessageStatus,
+} from '@tonlabs/uistory.chats';
+
+import type { SharingMessage } from './types';
+
+export function Sharing({ onLayout, text, onShare }: SharingMessage) {
+    return (
+        <View onLayout={onLayout}>
+            <BubbleSimplePlainText
+                type={ChatMessageType.PlainText}
+                key="sharing-bubble-text"
+                text={text}
+                status={MessageStatus.Received}
+                firstFromChain
+            />
+            <BubbleActionButton
+                type={ChatMessageType.ActionButton}
+                key="address-input-bubble-select-assets"
+                status={MessageStatus.Received}
+                text={uiLocalized.Browser.Sharing.Share}
+                onPress={onShare}
```

Comment:
To be honest I don't like that this is a "Share" button which allows to do anything we want...
It either should include Share logic internally or just build a customisable component from Surf itself with any button title and action for it.

@savelichalex what do you think? IMHO, the context is misleading..

Reply:
This is a consequence of browser ui and logic being split in two repos

Reply:
If want to keep it in UIKit, I'd make this component more generic, a.k.a `ActionWithDescription` :)

Reply:
Or just implement it fully from Surf. Having "Share" with not share makes no sense